README.md

```diff
@@ -201,6 +201,22 @@ or if using dockerized Devnet:
 $ docker run -e RUST_LOG=<LEVEL> shardlabs/starknet-devnet-rs
 ```
 
+By default, logging of request and response data is turned off.
+To see the request or/and response body, additional level have to be provided via `RUST_LOG` environment variable.
+To log the request body use `REQUEST`, to log the response body use `RESPONSE`.
+
+NOTE! that logging request and response requires at least logging level `INFO`.
+
+Example:
+
+```
+$ RUST_LOG="REQUEST, RESPONSE" starknet-devnet
+```
+
+```
+$ RUST_LOG="REQUEST, RESPONSE, INFO" starknet-devnet
+```
```

Comment:
Is it expected that there are no errors if providing an invalid rust_log value?

Reply:
we are using `parse_lossy` method, which gives warnings and ignores the invalid ones. But the new levels are being removed from the env_var string when the EnvFilter is being constructed